src/meshapi/views/forms.py

```diff
@@ -255,6 +293,48 @@ class NetworkNumberAssignmentRequest:
     install_number: int
 
 
+class NetworkNumberAssignmentRequestSerializer(DataclassSerializer):
+    class Meta:
+        dataclass = NetworkNumberAssignmentRequest
+
+    password = serializers.CharField()
+
+
+nn_form_success_schema = inline_serializer(
+    "NNFormSuccessResponse",
+    fields={
+        "detail": serializers.CharField(),
+        "building_id": serializers.IntegerField(),
+        "install_number": serializers.IntegerField(),
+        "network_number": serializers.IntegerField(),
+        "created": serializers.BooleanField(),
+    },
+)
+
+
+@extend_schema_view(
+    post=extend_schema(
+        tags=["User Forms"],
+        summary="Assign a network number to a given Install object. Used by the NN Assignment form",
+        request=NetworkNumberAssignmentRequestSerializer,
+        responses={
+            "200": OpenApiResponse(
+                nn_form_success_schema,
+                description="This install already has an NN, no action has been perfomed in the DB",
+            ),
+            "201": OpenApiResponse(
+                nn_form_success_schema,
+                description="This install's building did not already have an install. One has been created and assigned.",
```

Comment:
Huh? I think this might be a mis-copypasta. The NN assign form cannot create installs.

Reply:
Good catch, fixed